core/src/main/java/pl/agh/edu/actor/slider/SliderComponent.java

```diff
@@ -0,0 +1,71 @@
+package pl.agh.edu.actor.slider;
+
+import com.badlogic.gdx.graphics.g2d.NinePatch;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
+import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
+import com.badlogic.gdx.scenes.scene2d.utils.Drawable;
+import com.badlogic.gdx.utils.Align;
+
+import pl.agh.edu.actor.HotelSkin;
+
+public class SliderComponent extends Table {
+	private final float SLIDER_WIDTH = 600f;
+	private final float SLIDER_HEIGHT = 60f;
+	private final float HORIZONTAL_OUTER_PADDING = 50f;
+	private final float HORIZONTAL_INNER_PADDING = 30f;
+	private Label valueLabel;
+	private Slider slider;
+	private Skin skin;
+	private String suffix;
+
+	public SliderComponent(String name, String suffix, float min, float max, float step) {
+		Stack stack = new Stack();
+		add(stack).height(SLIDER_HEIGHT);
+		skin = HotelSkin.getInstance();
+		NinePatch texture = skin.getPatch("gray-100-border-round");
```

Comment:
też druga sprawa git by było to nazwać background albo coś w tym rodzaju a nie tekstura